### PR TITLE
People may want to do this on Cloud9

### PR DESCRIPTION
**Pull request by atolia**
kubectl proxy listen to default 8001 port but Cloud9 needs 8080 port and pressing Preview button

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by atolia** [comment]
there is another easier way: one just need to type 'kubectl config view' and get URL login and password for access API and dashboard

File: 02-path-working-with-clusters/201-cluster-monitoring/readme.adoc [hunk 1]
```diff
@@ -47,9 +47,9 @@ If you are using v1.8 or above, deploy the Dashboard using the following command
 
 Dashboard can be seen using the following command:
 
-    kubectl proxy
+    kubectl proxy --port 8080
 
-Now, Dashboard is accessible at http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/.
+Now, if you ran 'kubectl proxy' localy Dashboard is accessible at http://localhost:8080/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/. Otherwise, if you are in AWS Clou9 environment you sould click on 'Preview' to see dashboard in browser.
```
Comment (christopherhein) [inline 02-path-working-with-clusters/201-cluster-monitoring/readme.adoc]: Couple spelling mistakes, `localy` should be `locally` and `Clou9` should be `Cloud9` and `sould` should be `should`

**Comment by christopherhein** [comment]
@atolia are you able to make these changes?

**Comment by christopherhein** [comment]
I would imagine once `kubectl` has support for binding to none `localhost` addresses this should work as expected - https://github.com/kubernetes/kubernetes/pull/46517

**Comment by arun-gupta** [comment]
Conflicts need to be resolved here. Is it getting updated? @christopherhein @atolia

**Comment by christopherhein** [comment]
Closed due to the docs already being updated

**Pull request closed without merging**